datasets/sem_eval_2020_task_11/sem_eval_2020_task_11.py

```diff
@@ -0,0 +1,215 @@
+                                "Whataboutism,Straw_Men,Red_Herring",
+                            ]
+                        ),
+                    }
+                ),
+            }
+        )
+
+        return datasets.DatasetInfo(
+            # This is the description that will appear on the datasets page.
+            description=_DESCRIPTION,
+            # This defines the different columns of the dataset and their types
+            features=features,  # Here we define them above because they are different between the two configurations
+            # If there's a common (input, target) tuple from the features,
+            # specify them here. They'll be used if as_supervised=True in
+            # builder.as_dataset.
+            supervised_keys=None,
+            # Homepage of the dataset for documentation
+            homepage=_HOMEPAGE,
+            # License for the dataset if available
+            license=_LICENSE,
+            # Citation for the dataset
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager):
+        """Returns SplitGenerators."""
+        path_to_manual_file = os.path.abspath(os.path.expanduser(dl_manager.manual_dir))
+
+        keys = {}
+        for split in ["train", "dev", "test"]:
+            articles_path = os.path.join(path_to_manual_file, f"{split}-articles")
+            articles_files = glob.glob(os.path.join(articles_path, "*.txt"))
+            keys[split] = [os.path.splitext(os.path.basename(af))[0] for af in articles_files]
+
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                # These kwargs will be passed to _generate_examples
+                gen_kwargs={"data_dir": path_to_manual_file, "keys": keys["train"], "split": "train", "labels": True},
+            ),
+            datasets.SplitGenerator(
+                name=datasets.Split.TEST,
+                # These kwargs will be passed to _generate_examples
+                gen_kwargs={"data_dir": path_to_manual_file, "keys": keys["test"], "split": "test", "labels": False},
+            ),
+            datasets.SplitGenerator(
+                name=datasets.Split.VALIDATION,
+                # These kwargs will be passed to _generate_examples
+                gen_kwargs={"data_dir": path_to_manual_file, "keys": keys["dev"], "split": "dev", "labels": True},
+            ),
+        ]
+
+    def _generate_examples(self, data_dir, keys, split, labels):
+        """ Yields examples. """
+
+        # Get the main path for the articles
+        articles_path = os.path.join(data_dir, f"{split}-articles")
+
+        # If the split has labels, get the main paths for labels
+        if labels:
+            si_labels_dir = os.path.join(data_dir, f"{split}-labels-task-si")
+            tc_labels_dir = os.path.join(data_dir, f"{split}-labels-task-flc-tc")
+
+        # Iterate through each article
+        for id_, key in enumerate(keys):
```

Comment:
```suggestion
        for id_, key in enumerate(sorted(keys)):
```
we need the order to be deterministic here